SPEC.md

```diff
@@ -490,6 +491,10 @@ The App Container specification defines an HTTP-based metadata service for provi
 The ACE must provide a Metadata service on the address given to the applications via the `AC_METADATA_URL` [environment variable](#execution-environment).
 
 Clients querying any of these endpoints MUST specify the `Metadata-Flavor: AppContainer` header.
+Clients must also authenticate by use of a bearer token.
+The bearer token MUST be passed in by the ACE via `AC_METADATA_TOKEN` environment variable.
+Since the bearer token is used to securely identify a pod, it SHOULD have no fewer than 128 bits (size of UUID) bits of entropy.
```

Comment:
Just realised you repeated repeated a word.

Can we tweak it slightly to this:
> Since the bearer token is used to securely identify a pod, it SHOULD have no fewer than 128 bits of entropy (i.e. size of UUID).